misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -150,7 +151,17 @@ class MiskWebModule(private val config: WebConfig) : KAbstractModule() {
 
   @Provides @Singleton
   fun provideJettyThreadPool(): QueuedThreadPool {
-    return config.jetty_max_thread_pool_size?.let { QueuedThreadPool(it) } ?: QueuedThreadPool()
+    val maxThreads = config.jetty_max_thread_pool_size
+    val minThreads = Math.min(8, maxThreads)
+    val idleTimeout = 60_000
+
+    val threadPool = QueuedThreadPool(
+        maxThreads,
+        minThreads,
+        idleTimeout,
+        ArrayBlockingQueue(config.jetty_max_thread_pool_queue_size))
```

Comment:
OMG there’s two different classes:
```
org.eclipse.jetty.util.BlockingArrayQueue
java.util.concurrent.ArrayBlockingQueue
```
I think you’re choosing the right one, despite the fact that it’s different from Jetty’s default.